x-pack/filebeat/docs/inputs/input-httpjson.asciidoc

```diff
@@ -1103,7 +1106,7 @@ maximum wait time in between such requests. If `request.retry.max_attempts` is n
 [float]
 ==== `chain[].while.request`
 
-Please refer to <<request-parameters,request parameters>> for more information. Required.
+See  <<request-parameters,request parameters>> .
```

Comment:
This one has lost the _Required._ in the last commit. Is that correct?

Reply:
@adriansr yes I removed it intentionally cause it was looking very weird while actually viewing the doc.